src/usage_metrics/raw/github_nonpartitioned.py

```diff
@@ -0,0 +1,40 @@
+"""Extract non-partitioned data from Github logs.
+
+This includes stargazer and fork data, which when queried from Github returns data for
+the entire history of the repository.
+"""
+
+from typing import Literal
+
+import pandas as pd
+from dagster import (
+    AssetExecutionContext,
+    AssetsDefinition,
+    asset,
+)
+
+from usage_metrics.raw.github_partitioned import (
+    CUMULATIVE_METRIC_TYPES,
+    GithubExtractor,
+)
+
+
+def cumulative_metrics_extraction_factory(
+    metric: Literal[*CUMULATIVE_METRIC_TYPES],
+) -> AssetsDefinition:
+    """Create Dagster asset for each weekly-reported metric."""
+
+    @asset(
+        name=f"raw_github_{metric}",
+        tags={"source": "github_nonpartitioned"},
+    )
+    def _raw_github_logs(context: AssetExecutionContext) -> pd.DataFrame:
+        """Extract Github logs from daily files and return one weekly DataFrame."""
+        return GithubExtractor(metric=metric).extract(context)
```

Comment:
How is this grabbing only one week of data?

Reply:
Ah, this is an outdated description of the method - I've updated it.